schemas/v1.0/azure.yaml.json

```diff
@@ -261,8 +258,7 @@
                         },
                         "then": {
                             "required": [
-                                "project",
-                                "language"
+                                "project"
                             ],
                             "properties": {
                                 "docker": false
```

Comment:
Will lang: docker fit for declarative agents? Also for code based it can be our usual lang set?